middleware/archives_queries.py

```diff
@@ -8,7 +11,13 @@
 ]
 
 
-def archives_get_results(conn):
+def archives_get_results(conn: PgConnection) -> list[tuple[Any, ...]]:
+    """
+    Fetches results from the data_sources table based on specific conditions.
```

Comment:
would clarify this function is for pulling data sources for the automatic archives script that performs caching

Reply:
@maxachis can you add this to the docstring?

Reply:
Oh sorry, I see it now